apis/core/config/crd/bases/core.tanzu.vmware.com_capabilities.yaml

```diff
@@ -326,6 +326,321 @@ spec:
             - results
             type: object
         type: object
+    served: false
+    storage: false
```

Comment:
question: Just wanna double check that we only want v1alpha2 to be served and stored, correct ?

Reply:
Correct.